sdk/easm/azure-analytics-defendereasm/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Release History
```

Comment:
The directory name should match the artifact id. This should be in `sdk/easm/azure-analytics-defender-easm/` directory.